runtime/data/test_data_with_header_and_footer.csv

```diff
@@ -0,0 +1,11 @@
+header
```

Comment:
please explain me this 'header' thing, also why did you created new file with test data?